icu4j/main/classes/core/src/com/ibm/icu/impl/ZoneMeta.java

```diff
@@ -69,7 +69,7 @@ private static synchronized Set<String> getSystemZIDs() {
             systemZones = REF_SYSTEM_ZONES.get();
         }
         if (systemZones == null) {
-            Set<String> systemIDs = new TreeSet<String>();
+            Set<String> systemIDs = new TreeSet<>();
```

Comment:
Why are these changes here? They are unrelated and unneeded for this new API, right?

Reply:
By the standard Eclipse project setting. When you touch the file, it automatically optimize this to Java 7+ style.
We used to control coding style through Eclipse project file settings, but I know more people not using the env these days.